pkg/options/options.go

```diff
@@ -50,6 +51,7 @@ type KubeRouterConfig struct {
 
 func NewKubeRouterConfig() *KubeRouterConfig {
 	return &KubeRouterConfig{
+		CacheSyncTimeout:   5 * time.Minute,
```

Comment:
I think we should have a lower value here something like 1 minute

Reply:
Well, as far as I can see, 1 minute is really enough.